README.md

```diff
@@ -1,2 +1,183 @@
-# terraform-azure-backup
-Highly opinionated management of Kubernetes backup with velero on AKS (Azure Kubernetes Service)
```

Comment:
Why do you always use this phrase, since any kind of 'configuration scheme' or code is opinionated in one or another way? Looks like bloat for me.

Reply:
In my understanding, "*Highly* opinionated" means that you can't configure much. Like Prettier is highly opinionated, because it stands by its core principles and only allows for some minor customizations.